### PR TITLE
Add commmand to delete authorizer

### PR DESCRIPTION
**Pull request by uneet7**
A brief description of the changes in this PR:
Part of the issue: https://github.com/0chain/0chain/issues/1625
- Adds command to delete authorizer by id

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zwalletcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- 0chain:
- Other: ...

### CODE REVIEW

File: cmd/bridge-auth-delete.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	Short: "Delete ZCNSC authorizer by ID",
+	Long:  `Delete ZCNSC authorizer by ID`,
+	Args:  cobra.MinimumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+
+		var (
+			err error
+		)
+
+		payload := &zcncore.DeleteAuthorizerPayload{}
```
Comment (dabasov) [inline cmd/bridge-auth-delete.go]: we need authorizer id here, since txn will be executed with delegate wallet

**Review by dabasov — CHANGES_REQUESTED** [review]

File: cmd/bridge-auth-delete.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+			}
+		} else {
+			ExitWithError("Error: id flag is missing")
+		}
+
+		payload := &zcncore.DeleteAuthorizerPayload{
+			ID: ID,
+		}
+		fmt.Println(payload)
+		fmt.Println("Payload")
```
Comment (peterlimg) [inline cmd/bridge-auth-delete.go]: Please remove debug logs

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: cmd/bridge-auth-delete.go (hunk 1 above)
Comment (uneet7) [reply cmd/bridge-auth-delete.go]: Updated

File: cmd/bridge-auth-delete.go (hunk 2 above)
Comment (uneet7) [reply cmd/bridge-auth-delete.go]: Updated

**Review by peterlimg — APPROVED** [review]
lgtm

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | YES  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | feat/delete-authorizer-command  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**